code/modules/mob/living/silicon/computer.dm

```diff
@@ -1,30 +1,26 @@
 /mob/living/silicon
 	var/register_alarms = 1
 	var/datum/nano_module/alarm_monitor/all/alarm_monitor
-	var/datum/nano_module/atmos_control/atmos_control
-	var/datum/nano_module/crew_monitor/crew_monitor
 	var/datum/nano_module/law_manager/law_manager
-	var/datum/nano_module/power_monitor/power_monitor
 	var/datum/nano_module/rcon/rcon
+	var/obj/item/modular_computer/silicon/computer
```

Comment:
I'm pretty sure this needs to be cleaned up in Destroy.

Reply:
Isnt it dealt with in destroy?

Reply:
Since you added it, unless you added it to destroy, it doesn't.